views/components/Sidebar/SideNavigation.test.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
should this be part of the global setup?

Reply:
I don't think that would work, as we also have Node tests in Jest. We could likely set up a filter for `jsx` though.

Reply:
Something like this in `package.json` would work:
```json
    "test": "npm run test:node && npm run test:dom",
    "test:node": "jest server",
    "test:dom": "jest --env=jsdom --setupFilesAfterEnv jest-dom-config.js views",
```

Reply:
We would have to split up the jsx tests into their own directory, we have some regular tests in the views directory that use the node environment

Reply:
This should do it [db57df9](https://github.com/AMP-SCZ/dpdash/pull/458/commits/db57df9da37ccf3946371f5afab8705016b2841e)

Reply:
the script worked in github actions 🥂